### DISCUSSION TITLE
Can't get the TRANSMISSION_WEB_UI variable to work

### DISCUSSION THREAD

**Discussion by astaplet** [haugene/docker-transmission-openvpn#2881] [Q&A]
Hi there! I'm running this image in the container manager on a Synology device. I had been running on an old version of the container for several years in which I'd manually installed an alternate UI. I recently updated to the newest version of the image and can no longer get alternate UIs to install manually, but the environment variable discussed in the documentation doesn't work for me. I've copied and pasted it exactly and yet I always end up with the default web UI. Has anybody else encountered this issue?

**Comment by benyjr** [comment]
Can you paste how you are invoking TRANSMISSION_WEB_UI?

I use docker-compose and use the following line:

      - TRANSMISSION_WEB_UI=transmissionic

**Reply by astaplet** [reply]
I'm running it via the container manager on my synology, so all the environment variables are simply set in the UI as seen in this screenshot. Everything else works correctly, it's only the web ui that doesn't work.
<img width="1581" alt="Screenshot 2024-10-02 at 11 20 01 PM" src="https://github.com/user-attachments/assets/dd63e82e-f7d6-4e3c-8dbc-1ae65a52a808">

**Reply by benyjr** [reply]
Hi @astaplet. I tried using transmission-web-control via Portainer stacks and it's working for me.

The only conclusion I can make is that there is something wrong with Synology's docker install.
